jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/DefaultServletTest.java

```diff
@@ -504,6 +505,11 @@ public void testListingProperUrlEncoding() throws Exception
          *         `-- six
          */
 
+        try (Stream<Path> entries = Files.walk(docRoot))
+        {
+            entries.forEach(System.err::println);
+        }
```

Comment:
This test directory tree dump should be removed once we have our first green build